lang/MoarVM/Portfile

```diff
@@ -53,6 +53,11 @@ platform darwin 8 {
     patchfiles-append patch-build-setup-tiger.diff
 }
 
+if {${os.platform} eq "darwin" && ${os.major} >= 10} {
+    configure.args-append \
```

Comment:
Should be `<= 10` but we can fix this in the next version bump

Reply:
Thanks for the heads-up; fixed, and rev-bumped.